docs/metrics.md

```diff
@@ -17,6 +20,99 @@ for many cases; some examples:
 Emitting custom metrics with Argo is easy, but it's important to understand what makes a good Prometheus metric and the
 best way to define metrics in Argo to avoid problems such as [cardinality explosion](https://stackoverflow.com/questions/46373442/how-dangerous-are-high-cardinality-labels-in-prometheus).
 
+Metrics can be collected using the OpenTelemetry protocol or via prometheus compatible scraping.
+
+## Metrics configuration
+
+It is possible to collect metrics via the OpenTelemetry protocol or via prometheus compatible scraping. Both of these mechanisms can be enabled at the same time, which could be useful if you'd like to migrate from one system to the other. Using multiple protocols at the same time is not intended for long term use however.
+
+OpenTelemetry is the recommended way of collecting metrics, and the OpenTelemetry collector can export metrics to prometheus via [the prometheus remote write exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusremotewriteexporter/README.md).
+
+### OpenTelemetry protocol
+
+To enable the OpenTelemetry protocol you must set the environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` or `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT`. It will not default to the default value if left blank, instead no Open
```

Comment:
```suggestion
To enable the OpenTelemetry protocol you must set the environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` or `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT`.
It will not use the default value if left blank, instead no Open
```


- simplify, per k8s style guide
- prefer 1 sentence per line, per style guide
- incomplete sentence?